build.sh

```diff
@@ -154,7 +162,9 @@ fi
 if (( NUMARGS == 0 )) || hasArg librmm; then
     ensureCMakeRan
     echo "building librmm..."
-    cmake --build "${LIBRMM_BUILD_DIR}" -j${PARALLEL_LEVEL} ${VERBOSE_FLAG}
+    cmake --build "${LIBRMM_BUILD_DIR}" -j${PARALLEL_LEVEL} ${VERBOSE_FLAG} \
+        -DBUILD_TESTS=${BUILD_TESTS} \
+        -DBUILD_BENCHMARKS=${BUILD_BENCHMARKS}
```

Comment:
This should be move to the `ensureCMakeRan` function

Reply:
Why wasn't it there already? All I did was add arguments.

Reply:
Sorry, I mean the new arguments should be in `ensureCMakeRan` with the other arguments. This line is running the build. The CMake configuration is done in `ensureCMakeRan`

Reply:
Done.